src/WingetCreateCore/Common/GitHub.cs

```diff
@@ -226,6 +237,28 @@ private static string GetJwtToken(string gitHubAppPrivateKeyPem, int gitHubAppId
             };
 
             return Jose.JWT.Encode(payload, rsa, JwsAlgorithm.RS256);
+        }
+
+        private async Task<bool> CheckDuplicatePackageIdRecursive(string[] packageId, string path, int index)
+        {
+            if (index == packageId.Length)
+            {
+                return true;
+            }
+
+            var contents = await this.github.Repository.Content.GetAllContents(this.wingetRepoOwner, this.wingetRepo, path);
+
+            foreach (RepositoryContent content in contents)
+            {
+                if (string.Equals(packageId[index].ToLowerInvariant(), content.Name.ToLowerInvariant()))

```

Comment:
>packageId[index].ToLowerInvariant() [](http://example.com/codeflow?start=34&length=35)

you can make this a variable above the loop so you only calculate it once

Reply:
Changed to suggestion